src/styles/card.css

```diff
@@ -0,0 +1,106 @@
+/* #region card */
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+/* .image {
+
+} */
+
+.card__product-name {
+  width: 166px;
+  height: 36px;
```

Comment:
do not hardcode width and height for content
it should be 100% width (you can add horizontal paddings for whole card)
and height should be defined by content, not hardcode